frontend/src/components/Banner.js

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+
+import React from "react";
+import HeaderTypography from "./HeaderTypography";
+import ParagraphTypography from "./ParagraphTypography";
+import { Link } from "react-router-dom";
+import RoundedEdgesButton from "./RoundedEdgesButton";
+
+import { textColor, activeTextColor } from "../assets/jss/colorPalette";
+import bannerImage from "../assets/img/banner.png";
+
+/**
+ * Displays a welcoming background image with a button.
+ * @param user user object of the currently logged in account
+ * @returns {JSX.Element}
+ */
+export default function Banner({ children, title, user }) {
+	const bannerTitle = <BannerTitle>{ title }</BannerTitle>;
+	const subTitle = <BannerSubTitle>{ children }</BannerSubTitle>;
+	const button = <BannerButton isLoggedIn={(user != null)} style={{ fontSize: "17px", }}/>;
+
+	const containerStyle = ({
+		height: "305px",
+	});
+	const backgroundImageStyle = ({
+		padding: "44px",
+		alignItems: "center",
+		justifyContent: "center",
+		textAlign: "center",
+	
+		borderRadius: "15px",
+		backgroundImage: `url(${bannerImage})`,
+		backgroundPosition: "center",
+		backgroundSize: "cover",
+	});
+
+	const BannerItem = ({ children, style }) => {
+		const itemStyle = ({
+			width: "100%",
+			display: "flex",
+			alignItems: "center",
+			justifyContent: "center",
+			...style,
+		});
+		return <div style={itemStyle}>{ children }</div>;
+	};
+
+	return (
+		<div style={containerStyle}>
+			<div style={backgroundImageStyle}>
+				<BannerItem style={{ marginBottom: "11px" }}>{ bannerTitle }</BannerItem>
+				<BannerItem style={{ height: "66px", marginBottom: "32px" }}>{ subTitle }</BannerItem>
+				<BannerItem style={{ }}>{ button }</BannerItem>
+			</div>
+		</div>
+	);
+}
+
+const newBannerStyle = (style) => ({ color: textColor, ...style, });
+
+function BannerTitle({ style, children }) {
+	const smallStyle = newBannerStyle({ fontSize: 30, fontWeight: 550, bottomMargin: "5px", ...style });
+	const mediumStyle = newBannerStyle({ fontSize: 37, fontWeight: 700, ...style });
+
+	return <HeaderTypography sx={smallStyle} md={mediumStyle}>{ children }</HeaderTypography>;
+}
+
+function BannerSubTitle({ style, children }) {
+	const smallStyle = newBannerStyle({ fontSize: 19, fontWeight: 400, ...style });
+	const mediumStyle = newBannerStyle({ fontSize: 22, fontWeight: 500, ...style });
+
+	return <ParagraphTypography sx={smallStyle} md={mediumStyle}>{ children }</ParagraphTypography>;
+}
+
+// according to mockup, the Banner's Button is slightly larger than the Header's
+function BannerButton({ isLoggedIn, style }) {
+	const buttonStyle = ({
+		height: "54px",
+		width: "192px",
+		fontWeight: 700,
+		...style,
+	});
+
+	if (!isLoggedIn) {
+		return <SignupButton style={buttonStyle}/>;
+	}
+	return <ProfileButton style={buttonStyle}/>;
```

Comment:
I think something like the following would look nicer. But can also stay the way it is.
```JavaScript
let BannerButtonComponent = isLoggedIn ? ProfileButton : SignupButton;
return <BannerButtonComponent style={buttonStyle}/>;
```